### PR TITLE
Add testing for sequence and ensemble models

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: qa/L0_perf_analyzer_capi/test.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ fi
 export CUDA_VISIBLE_DEVICES=0
 
 CLIENT_LOG="./perf_analyzer.log"
-PERF_ANALYZER=../clients/perf_analyzer
+PERF_ANALYZER=/opt/tritonserver/bin/perf_analyzer
```
Comment (rmccorm4) [inline qa/L0_perf_analyzer_capi/test.sh:45]: is this intended? will it use the most recently built PA during CI?

File: qa/L0_perf_analyzer_capi/test.sh [hunk 2]
```diff
@@ -74,7 +74,7 @@ cp -r /data/inferenceserver/${REPO_VERSION}/qa_variable_model_repository/graphde
 cp -r /data/inferenceserver/${REPO_VERSION}/qa_variable_model_repository/graphdef_int32_int32_float32 $DATADIR/
 
 # Copy shape tensor models
-cp -r /data/inferenceserver/${REPO_VERSION}/qa_shapetensor_model_repository/plan_zero_1_float32 $DATADIR/
+# cp -r /data/inferenceserver/${REPO_VERSION}/qa_shapetensor_model_repository/plan_zero_1_float32 $DATADIR/
```
Comment (rmccorm4) [inline qa/L0_perf_analyzer_capi/test.sh:77]: is this unused?

File: qa/L0_perf_analyzer_capi/test.sh (hunk 1 above)
Comment (Tabrizian) [reply qa/L0_perf_analyzer_capi/test.sh]: Good catch! Fixed.

File: qa/L0_perf_analyzer_capi/test.sh (hunk 2 above)
Comment (Tabrizian) [reply qa/L0_perf_analyzer_capi/test.sh]: Good catch. It has been fixed.

**Review by debermudez — DISMISSED** [review]

**Review by rmccorm4 — APPROVED** [review]

**Review by tanmayv25 — DISMISSED** [review]

**Review by matthewkotila — APPROVED** [review]

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**